modules/ROOT/nav.adoc

```diff
@@ -2,6 +2,8 @@ Couchbase PHP SDK
 
 .Getting Started!
 * xref:hello-world:start-using-sdk.adoc[Start Using the PHP SDK]
+* xref:hello-world:platform-help.adoc[PHP Platform Help]
```

Comment:
Were't we putting these two in project-docs?
That's what I've done in Python after our discussion.

Reply:
Well, it needn't move out of the hello-world module,
but it's outside of the quickstart journey,
so lives alongside the full installation docs in project-docs (see them there in Python 4.0)

Reply:
Yes, will check against Python 👍  looks like I got it precisely the wrong way around?